.golangci.yml

```diff
@@ -21,7 +21,7 @@ linters-settings:
     lines: 100
     statements: 50
   gci:
-    local-prefixes: github.com/mesosphere/konvoy-image-builder
+    prefixes: github.com/mesosphere/konvoy-image-builder
```

Comment:
Shouldn't this config be more like https://golangci-lint.run/usage/linters/#gci ?

Reply:
fixed